examples/prometheus-operator/README.md

```diff
@@ -0,0 +1,55 @@
+# How to monitor TiDB using prometheus-operator
+
+> **Note:**
+>
+> This example describe ServiceMonitor and PrometheusRule yaml demo , you can use existing Prometheus CRD instance to load .
+
+**Prerequisites**: 
+- Has Prometheus operator installed. [Doc](https://github.com/coreos/kube-prometheus)
+
+  This could by verified by the following command:
+  
+  ```bash
+  >  kubectl get crd |grep coreos.com
+  ```
+  
+  The output is similar to this:
+  
+  ```bash
+        NAME                                       TIME
+    alertmanagers.monitoring.coreos.com            2020-04-12T14:22:49Z
+    podmonitors.monitoring.coreos.com              2020-04-12T14:22:50Z
+    prometheuses.monitoring.coreos.com             2020-04-12T14:22:50Z
+    prometheusrules.monitoring.coreos.com          2020-04-12T14:22:50Z
+    servicemonitors.monitoring.coreos.com          2020-04-12T14:22:51Z
+  ```
+  
+## Initialize
+
+Initialize serviceMonitor with label ` app: tidb cluster: basic` and prometheus rules.
```

Comment:
We should tell users that this case is only used to demo and not suggest to use in the prod env.